database/src/migrations/20250225000001_rename_environment_unit_enum.ts

```diff
@@ -0,0 +1,27 @@
+import { Knex } from 'knex';
+
+/**
+ * Rename the `environment_unit` enum to `quantitative_unit`.
+ *
+ * Originally created in migration `20240417000000_obsevation_environment_tables.ts` when only the environment tables
```

Comment:
mega nit: `_obsevation_` -> `_observation_`

ignore if thats the actual name of the file

Reply:
Fixed